WrightTools/data/_spcm.py

```diff
@@ -74,19 +68,61 @@ def from_spcm(filepath, name=None, *, delimiter=",", format=None, parent=None, v
                     headers["resolution"] = int(value.strip(" bits ADC"))
                 else:
                     headers[key.strip()] = value.strip()
+        line = f.readline().strip()
+        while "_BEGIN" in line:
+            header_lines += 1
+            section = line.split("_BEGIN")[0]
+            while True:
+                line = f.readline().strip()
+                header_lines += 1
+                if section + "_END" in line:
+                    break
+                if section == "SYS_PARA":
+                    use_type = {
+                        "B": lambda b: int(b) == 1,
+                        "C": str,  # e.g. #SP [SP_OVERFL,C,N]
+                        "F": float,
+                        "I": int,
+                        "L": int,  # e.g. #DI [DI_MAXCNT,L,128]
+                        "S": str,
+                        "U": int,  # unsigned int?
+                    }
+                    item = line[line.find("[") + 1 : line.find("]")].split(",")
+                    key = item[0]
+                    value = use_type[item[1]](item[2])
+                    headers[key] = value
+                else:
+                    splitted = line.split()
+                    value = splitted[-1][1:-1].split(",")
+                    key = " ".join(splitted[:-1])
+                    headers[key] = value
+            line = f.readline().strip()
+            if "END" in line:
+                header_lines += 1
+                break
+    # initialize data object
+    kwargs = {"name": name, "kind": "spcm", "source": filepath, **headers}
```

Comment:
If we have access to the time of collection, it is intended behavior that the data object "created" date use that time. See `_pycmds.py` for an example of such. (Just a matter of passing in the kwarg/ensuring the date format is what we want) 

Here, it seems to be the combination of two header fields "Date" and "Time"

Reply:
What timezone would the investment be in?